azerothcore/azerothcore-wotlk
# fix(Core/GridNotifiers): fix tempsummons not finding nearby creatures

## Changes Proposed:
-   fix tempsummons not finding nearby creatures, fixes condition type 29
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9726
- Closes https://github.com/chromiecraft/chromiecraft/issues/2589

## SOURCE:


## Tests Performed:

- Tested in-game
- 


## How to Test the Changes:


Apply the SQL provided
Try summoning Mor/Valathak before **and** after compiling your core with this PR
Notice the creature check always fails (nothing ever spawns) without this PR

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
